README.md

```diff
@@ -1,3 +1,194 @@
+      "text": "intro.md",
+      "code": "env-init.sh",
+      "credits": ""
+    },
+    "steps": [
+      {
+        "title": Title for the step.
+        "text": Filename containing the body for the step.
+        "answer": Filename containing the answer for the step.
+      },
+      {
+        "title": Title for the step.
+        "text": Filename containing the body for the step.
+        "answer": Filename containing the answer for the step.
+      }
+    ],
+    "finish": {
+      "text": "finish.md"
+    }
+  },
+  "files": ["index.html", "myapp.scss"],
+  "environment": {
+    "showdashboard": true,
+    "delayToSaveFileAfterStopTypingMilliseconds": 2000,
+    "autoFormatEnable": false,
+    "dashboards": [
+      {"name": "Web Application", "port": 3000}
+    ],
+    "uilayout": "editor-iframe-split",
+    "uisettings": html or javascript
+  },
+  "backend": {
+    "imageid": "node",
+    "port": 3000
+  }
+}
+```
+
+## Content and steps
+
+When writing out the steps for your tutorial please follow these guidelines:
+
+### Name of module
+
+* Purpose: To label modules. Name of modules should be succinct and use sentence case.
+    * Incorrect: Design, develop, and implement toolbar component with a filter.
+    * Correct: Toolbar component with filter.
+
+### Description
+
+* Purpose: To accurately describe the purpose of the module. The description should be 1--2 sentences. Avoid "you" or "we".
+    * Incorrect: You will build a React table with pagination.
+    * Correct: Build a React table with pagination.
+
+### Difficulty level
+
+* Purpose: To communicate the difficulty level of the module to the user.
+    * Beginner: For those new to PatternFly or new to a concept in PatternFly.
+    * Intermediate: For those familiar with PatternFly. Assumes that a user has some prior experience with PatternFly.
+    * Advanced: For regular PatternFly users. Modules involve higher level PatternFly React and PatternFly CSS concepts.
+
+### Estimated time
+
+* Purpose: To estimate the time that it will take the user to complete the module.
+
+### Introduction page
+
+* Purpose: To introduce the user to the module content. The introduction is always a critical part of any course. It is important to be brief, but clear about what information will be covered. The approximate length should be 3-5 sentences.
+    * Incorrect: PatternFly React charts are fun to use. (Not enough information about the module's content).
+    * Correct: PatternFly consists of isolated and modular structures that fall into three categories: Components: Components are modular and independent structures concerned with presentation. Layouts: Layouts allow for organizing and grouping their immediate children on the page. Demos: Demos illustrate how to assemble complex structures with components and layouts only. Utilities: Utilities are a set of classes that enable you to further customize and modify elements in your project without having to write any custom CSS.
```

Comment:
```suggestion
    * Correct: PatternFly consists of isolated and modular structures that fall into three categories: Components: Components are modular and independent structures concerned with presentation. 
  
- Layouts: Layouts allow for organizing and grouping their immediate children on the page. 
- Demos: Demos illustrate how to assemble complex structures with components and layouts only. 
-  Utilities: Utilities are a set of classes that enable that customize and modify elements in the module without having to write any custom CSS.

```

Reply:
we mentioned to use sentences here so why would we break them into bullets?